src/core/Navigation/WizardNavigation/WizardNavigation/WizardNavigation.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactNode, useState } from 'react';
+import { HtmlButton, HtmlDiv, HtmlNav, HtmlUl } from '../../../../reset';
+import styled from 'styled-components';
+import { SuomifiThemeConsumer } from '../../../theme';
+import { baseStyles } from './WizardNavigation.baseStyles';
+import classnames from 'classnames';
+import { Icon } from '../../../Icon/Icon';
+
+export interface WizardNavigationProps {
+  children: ReactNode | ReactNode[];
+  /**
+   * Normal or small screen variant
+   * @default normal
+   */
+  variant?: 'default' | 'smallScreen';
+  /**
+   * Whether the menu is initially expanded. Only applies to smallScreen variant.
+   * @default true
+   */
+  initiallyExpanded?: boolean;
```

Comment:
Since the `initiallyExpanded` only applies to the smallScreen variant, should this be a conditional prop? Probably not a big deal but worth a discussion I think.

Reply:
Fair point, but as far as I understand the only benefit you would get out of it is Typescript not accepting the `initiallyExpanded` prop if the variant !== smallScreen. So not a huge thing IMO

Reply:
Yeah, it's more of a matter of principle with these. And judging from past experiences this more strict typing can be more of an annoyance for the users. So I'm fine with keeping it as it is.